Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -420,42 +420,42 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
                 $fixerDetailLine = false;
                 if (OutputInterface::VERBOSITY_VERBOSE <= $verbosity) {
-                    $fixerDetailLine = $output->isDecorated() ? ' (<comment>%s</comment>)' : ' %s';
+                    $fixerDetailLine = $stdErr->isDecorated() ? ' (<comment>%s</comment>)' : ' %s';
                 }
 
                 foreach ($changed as $file => $fixResult) {
-                    $output->write(sprintf('%4d) %s', $i++, $file));
+                    $stdErr->write(sprintf('%4d) %s', $i++, $file));
 
                     if ($fixerDetailLine) {
-                        $output->write(sprintf($fixerDetailLine, implode(', ', $fixResult['appliedFixers'])));
+                        $stdErr->write(sprintf($fixerDetailLine, implode(', ', $fixResult['appliedFixers'])));
+                        $stdErr->writeln('');
                     }
 
                     if ($input->getOption('diff')) {
                         $output->writeln('');
                         $output->writeln('<comment>      ---------- begin diff ----------</comment>');
                         $output->writeln($fixResult['diff']);
                         $output->writeln('<comment>      ---------- end diff ----------</comment>');
+                        $output->writeln('');
                     }
-
-                    $output->writeln('');
                 }
 
                 if (OutputInterface::VERBOSITY_DEBUG <= $verbosity) {
-                    $output->writeln('Fixing time per file:');
+                    $stdErr->writeln('Fixing time per file:');
 
                     foreach ($this->stopwatch->getSectionEvents('fixFile') as $file => $event) {
                         if ('__section__' === $file) {
                             continue;
                         }
 
-                        $output->writeln(sprintf('[%.3f s] %s', $event->getDuration() / 1000, $file));
+                        $stdErr->writeln(sprintf('[%.3f s] %s', $event->getDuration() / 1000, $file));
                     }
 
-                    $output->writeln('');
+                    $stdErr->writeln('');
                 }
 
                 $fixEvent = $this->stopwatch->getEvent('fixFiles');
-                $output->writeln(sprintf('%s all files in %.3f seconds, %.3f MB memory used', $input->getOption('dry-run') ? 'Checked' : 'Fixed', $fixEvent->getDuration() / 1000, $fixEvent->getMemory() / 1024 / 1024));
+                $stdErr->writeln(sprintf('%s all files in %.3f seconds, %.3f MB memory used', $input->getOption('dry-run') ? 'Checked' : 'Fixed', $fixEvent->getDuration() / 1000, $fixEvent->getMemory() / 1024 / 1024));
```

Comment:
It is not, the naming of the streams is confusing. It is better, IMO, to send all kind of information to stdErr and only parsable formatted info to stdOut